### PR TITLE
commit change

### PR DESCRIPTION
**Pull request by HninNweOo**
added text file

### CODE REVIEW

**Comment by HninNweOo** [comment]
@pyonemyatkhine  review the program

**Comment by HninNweOo** [comment]
@pyonemyatkhine @aungkyaw92 @Toshi7  review the program

File: exercise.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (HninNweOo) [inline exercise.txt:1]: jjjjjj

Comment (pyonemyatkhine) [reply exercise.txt:1]: ***\* Review here *****

**Pull request merged**